### PR TITLE
allow organization update for user profile

### PR DESCRIPTION
**Pull request by CHRISTOPHERKADOGO**
# Description

This PR Allows to update organization for users, and also make patch endpoint flexible to cater for updating individual properties about the user

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## ClickUp Ticket ID
https://app.clickup.com/t/869542h0p

## How Can This Be Tested?
Run this  branch  locally and try updating the properties about user (name, is public, organization)

### CODE REVIEW

File: app/controllers/users.py [hunk 1]
```diff
@@ -429,8 +430,8 @@ def patch(self, user_id):
         """
         """
         try:
-            user_schema = UserSchema(only=("name", "is_public"))
-
+            
+            user_schema = UserUpdateSchema()
```
Comment (rhodinemma) [inline app/controllers/users.py]: Maybe instead of using a new schema, we could use the `Partial=true` on the existing schema.

**Review by rhodinemma — APPROVED** [review]
Thanks for this @CHRISTOPHERKADOGO

File: app/controllers/users.py [hunk 2]
```diff
@@ -429,8 +429,8 @@ def patch(self, user_id):
         """
         """
         try:
-            user_schema = UserSchema(only=("name", "is_public"))
-
+            
+            user_schema = UserSchema(partial=True)
```
Comment (LanternNassi) [inline app/controllers/users.py]: Thanks for this Chris . Though this poses some security concerns on the endpoint as this means someone can modify fields such as `is_beta_user` , `verified `  once they can bundle them with the payload which are only to be updated under certain circumstances . 

```suggestion
            user_schema = UserSchema(only=("name", "is_public" , "organisation") , partial=True)
```

**Review by LanternNassi — CHANGES_REQUESTED** [review]
Great Work on this

**Review by LanternNassi — APPROVED** [review]
LGTM . Nice work ..

**Pull request merged**